backend/canisters/user_index/impl/src/updates/c2c_notify_events.rs

```diff
@@ -23,17 +23,15 @@ fn c2c_notify_events_impl(args: Args, runtime_state: &mut RuntimeState) -> Respo
 fn handle_event(event: Event, runtime_state: &mut RuntimeState) {
     match event {
         Event::UserJoinedGroup(ev) => {
-            if let Some(index) = runtime_state.data.local_index_map.get_index_canister(&ev.user_id) {
-                runtime_state.data.user_index_event_sync_queue.push(
-                    index,
-                    LocalUserIndexEvent::UserJoinedGroup(UserJoinedGroup {
-                        user_id: ev.user_id,
-                        chat_id: ev.chat_id,
-                        as_super_admin: ev.as_super_admin,
-                        latest_message_index: ev.latest_message_index,
-                    }),
-                );
-            }
+            runtime_state.push_event_to_local_user_index(
```

Comment:
The issue was that here the job wasn't being started, but now it is because the method on `RuntimeState` starts the job if needed.